assets/js/TableView.js

```diff
@@ -104,7 +108,8 @@ export default class TableView extends React.Component {
         rowGetter={this.getRow}
         rowsCount={this.props.totalRows }
         minWidth={this.state.gridWidth -2}
-        minHeight={this.state.gridHeight-2} />;   // -1 because grid has borders, don't want to expand flex grid
+        minHeight={this.state.gridHeight-2}
+        onRowClick={this.props.onClick} />;   // -1 because grid has borders, don't want to expand flex grid
```

Comment:
This seems not to be passed in where TableView is created in OutputPane -- still needed? If you want to keep it, then, onClick should be in propTypes.